dev/customer_testing/run_tests.dart

```diff
@@ -25,6 +25,18 @@ Future<bool> run(List<String> arguments) async {
       help: 'How many times to run each test. Set to a high value to look for flakes.',
       valueHelp: 'count',
     )
+    ..addOption(
+      'shards',
+      defaultsTo: '1',
+      help: 'How many shards to split the tests into. Used in continuous integration.',
+      valueHelp: 'count',
+    )
+    ..addOption(
+      'shard-index',
+      defaultsTo: '0',
+      help: 'The current shard to run the tests as. Used in continuous integration.',
```

Comment:
it should say what the valid range is (presumably 0..shard-1 ?)

Reply:
Added. I refrained initially as all the other help lines were short :)